ouroboros-network/src/Ouroboros/Network/Testing/ConcreteBlock.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE StandaloneDeriving         #-}
 {-# LANGUAGE TypeFamilies               #-}
 {-# LANGUAGE UndecidableInstances       #-}
+{-# LANGUAGE FlexibleContexts           #-}
```

Comment:
This is due to a bug fix in 9.2:

> Previously, `-XUndecidableInstances` accidentally implied `-XFlexibleContexts`. This is now fixed, but it means that some programs will newly require `-XFlexibleContexts`.

https://downloads.haskell.org/~ghc/9.2.4/docs/html/users_guide/9.2.1-notes.html

Reply:
Thanks for pointing this out!